### DISCUSSION TITLE
How to do HLS?

### DISCUSSION THREAD

**Discussion by z0rti** [videojs/video.js#8470] [General]
### Description

Not a bug but i wonder how to use HLS with [https://github.com/arut/nginx-rtmp-module](https://github.com/arut/nginx-rtmp-module)
I set it up to create files in hls/test.m3u8, works in mpv/vlc
Is it that simple?

```
<!DOCTYPE html>
<html lang="en">
<head>

    <title>Video.js | HTML5 Video Player</title>
    <link href="http://vjs.zencdn.net/7.0/video-js.min.css" rel="stylesheet">
    <script src="http://vjs.zencdn.net/7.0/video.min.js"></script>

</head>
<body>

  <video id="example_video_1" class="video-js" controls preload="none" width="640" height="360" poster="hls.png" data-setup="{}">
    <source src="hls/test.m3u8" type="application/x-mpegURL">
    <p class="vjs-no-js">To view this video please enable JavaScript, and consider upgrading to a web browser that <a href="https://videojs.com/html5-video-support/" target="_blank">supports HTML5 video</a></p>
  </video>

</body>

</html>
```
I tested it, but it seems to have problems, buffering, trying to play old segments that are gone

### Reduced test case

http://no.ne

### Steps to reproduce

1. install nginx-rtmp-module
2. in web dir: git clone https://github.com/videojs/video.js.git
3. setup to point to hls m3u8

### What version of Video.js are you using?

Video.js 8.6.0

### What browser(s) including version(s) does this occur with?

firefox 118

### What OS(es) and version(s) does this occur with?

Linux

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! 👋

If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md).

**Comment by krustyth88** [comment]
Hello, how can I integrate the user agent of an m3u8 to the player, I have tried it and it does not play, or call it from a json.
